utils/s3/retry_strategy.cc

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "retry_strategy.hh"
+#include "aws_error.hh"
+
+namespace aws {
+default_retry_strategy::default_retry_strategy(uint32_t max_retries, uint32_t scale_factor) : _max_retries(max_retries), _scale_factor(scale_factor) {
+}
+
+retryable default_retry_strategy::should_retry(const aws_error& error, uint32_t attempted_retries) const {
+    if (attempted_retries >= _max_retries)
+        return retryable{false};
```

Comment:
please add braces around this single-line statement. for better readability.

Reply:
Done